src/components/WarningHighlight.js

```diff
@@ -1,15 +1,20 @@
-import { h } from 'preact';
-import WarningTooltip from './WarningTooltip.js';
+import React from 'react';
+import { Tooltip } from 'react-tooltip';
 
 export default function Highlight(props) {
   return (
     <div
       className="jns-highlight"
       style={props.styles}
-      data-tip={props.message}
-      data-place={props.position}
+      data-tooltip-content={props.message}
+      data-tooltip-place={props.position}
+      data-tooltip-position-strategy="fixed"
     >
-      <WarningTooltip />
+      <Tooltip
+        className="jns-tooltip"
+        anchorSelect=".jns-highlight"
+        float={true}
+      />
```

Comment:
In GMail, the message for "does this make sense" is overflowing the box:

![image](https://user-images.githubusercontent.com/1520763/220695830-4a95ddaa-f29f-4c1d-9952-e9656d604a90.png)

Maybe get rid of `max-height` from jns-tooltip CSS?

Reply:
Actually, the white text is there, but the background color doesn't extend all of the way down:
![image](https://user-images.githubusercontent.com/1520763/220696988-9e0271cb-7519-4057-9d7d-f060d325d1fd.png)